wire-runtime-swift/build.gradle.kts

```diff
@@ -69,6 +69,24 @@ val generateSwiftProtos by tasks.creating(JavaExec::class) {
   }
 }
 
+val generateSampleProtos by tasks.creating(JavaExec::class) {
```

Comment:
There's almost certainly a better way to do this but I don't know gradle

Reply:
@oldergod 

> Why not add those files in the existing generateTestProtos directly?

I think there's value in a different subdirectory, tho it for sure should just be happening as a dependency or something. 

I am very unfamiliar with Gradle; my only experience with it was XML _years_ ago.

Reply:
Added as a dependency